control-plane/cmd/post-install/kafka_source_dispatcher_deleter.go

```diff
@@ -38,3 +44,17 @@ func (d *kafkaSourceDeleter) DeleteDispatcher(ctx context.Context) error {
 
 	return nil
 }
+
+func (d *kafkaSourceDeleter) waitStatefulSetExists(ctx context.Context) error {
+	const sourceDispatcherStatefulSetName = "kafka-source-dispatcher"
```

Comment:
There's also a declaration of the same constant on [line 36](https://github.com/knative-sandbox/eventing-kafka-broker/pull/2667/files#diff-60bca8e7937b5364cdc18d6afe44c8e11e5df784e6a155014a1e8574f63a8b8bR36). Maybe we declare one outside or pass name as func input?

Reply:
I separated them because, in theory, they can have different names since one is the statefulset and one is the deployment.

Reply:
Ok.